dotnetapp-preview/README.md

```diff
@@ -1,25 +1,41 @@
 dotnetapp-preview Sample
 ========================
 
-The dotnetapp-preview sample demonstrates basic "hello world" usage of .NET Core. It shows you how you can build and deploy it relying on a preview version of the .NET Core SDK. It is the same as the [dotnetapp-dev](../dotnetapp-preview) sample, but has been updated to require a later version of the .NET Core SDK and depend on a Docker image from the [dotnet-nightly](https://hub.docker.com/r/microsoft/dotnet-nightly/) Docker Hub repo. This repo is where preview .NET Core versions are published.
+The dotnetapp-preview sample demonstrates how you can build and run the dotnetapp sample using the [.NET Core Runtime 1.1.0-preview1 Docker image](https://hub.docker.com/r/microsoft/dotnet/). It's a great option for trying the .NET Core 1.1 version before it is released. It is the same as the [dotnetapp-prod](../donetapp-prod) image, except that it relies on a later .NET Core version.
 
-You don't even need .NET Core installed on your local machine to use this sample. It's a great way to get started with .NET Core. It's also a good option for iterative development if you want to use a container as your development environment.
+The instructions assume that you already have [.NET Core](https://dot.net/core) and [Git](https://git-scm.com/downloads) and [Docker](https://www.docker.com/products/docker) clients installed. They also assume you already know how to target Linux or Windows containers. Do try both image types. You need the latest Windows 10 or Windows Server 2016 to use [Windows containers](http://aka.ms/windowscontainers).
```

Comment:
>[.NET Core](https://dot.net/core) [](start = 46, length = 33)

Specifically don't you need to have the 1.1 shared framework installed as well locally in order to restore?  Also do you need a NuGet.config to point to the myget feed?